app/controllers/DigitalSubscription.scala

```diff
@@ -61,14 +61,18 @@ class DigitalSubscription(
     val css = "digitalSubscriptionLandingPage.css"
     val description = stringsConfig.digitalPackLandingDescription
     val canonicalLink = Some(buildCanonicalDigitalSubscriptionLink("uk"))
+    val promoCode = request.queryString.get("promoCode").flatMap(_.headOption)
     val hrefLangLinks = Map(
       "en-us" -> buildCanonicalDigitalSubscriptionLink("us"),
       "en-gb" -> buildCanonicalDigitalSubscriptionLink("uk"),
       "en-au" -> buildCanonicalDigitalSubscriptionLink("au"),
       "en" -> buildCanonicalDigitalSubscriptionLink("int")
     )
+    val productPrices = priceSummaryServiceProvider.forUser(false).getPrices(DigitalPack, promoCode)
```

Comment:
You need to pass in true for test users here

Reply:
@rupertbates I don't have an iduser on this route :(

Reply:
You can still check whether they have a `_test_username` cookie. Have a look at TestUserService

Reply:
oh!